tests/rnd/00-test-battery-chunking.t

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test the cylc test-battery --chunk option
```

Comment:
Very meta.

Reply:
Perhaps we should test the testing of this test?

Reply:
![image](https://user-images.githubusercontent.com/2362137/43393330-bfccdc00-944a-11e8-89fe-a7de0dc8741d.png)

Reply:
![foo](http://www.funnysigns.net/files/nothing-happened-400x400.jpg)